pkgs/os-specific/linux/wpa_supplicant/default.nix

```diff
@@ -3,28 +3,19 @@
 , dbusSupport ? !stdenv.hostPlatform.isStatic, dbus
 , withReadline ? true, readline
 , withPcsclite ? !stdenv.hostPlatform.isStatic, pcsclite
-, readOnlyModeSSIDs ? false
 }:
 
 with lib;
 stdenv.mkDerivation rec {
-  version = "2.10";
+  version = "2.11";
 
   pname = "wpa_supplicant";
 
   src = fetchurl {
     url = "https://w1.fi/releases/${pname}-${version}.tar.gz";
-    sha256 = "sha256-IN965RVLODA1X4q0JpEjqHr/3qWf50/pKSqR0Nfhey8=";
+    sha256 = "sha256-kS6gb3TjCo42+7aAZNbN/yGNjVkdsPxddd7myBrH/Ao=";
   };
 
-  patches = [
-    # Fix a bug when using two config files
-    ./Use-unique-IDs-for-networks-and-credentials.patch
-  ] ++ lib.optionals readOnlyModeSSIDs [
-    # Allow read-only networks
-    ./0001-Implement-read-only-mode-for-ssids.patch
-  ];
```

Comment:
I'm not aware why these two patches existed before, but I cannot find anything related in the release notes. Are both of these upstreamed now?

Reply:
Yes, both are upstream:
- unique IDs: http://w1.fi/cgit/hostap/commit/?id=0adc67612dab5a81088a832b1fd9629d3a2cada8
- read only mode for ssids: http://w1.fi/cgit/hostap/commit/?id=d8d2b3a338a18e5f3f5ef82b7279b7c082633f94

Reply:
Awesome

Reply:
Perfect!